packages/components/psammead-story-promo-list/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 6.0.21 | [PR#4547](https://github.com/bbc/psammead/pull/4547) Bumps packages |
```

Comment:
This should bump to version 6.0.22, I am sure it is just a conflict with some new releases.